description.md

```diff
@@ -34,6 +34,50 @@ Put differently, it shows how each content type is to be used in the various tea
 
 ### Practical Sessions
 
+Practical sessions are represented by hands-on activities that learners perform in order to acquire practical experience.
+These activities are typically supervized by a teacher that proposes the practical exercices and evaluates the learner's performance.
+The purpose of the evaluation is mainly to provide constructive feedback, as opposed to grading the student.
+
+Instances of practical sessions are represented by laboratories, seminars or workshops.
+The format of a practical session may include:
+
+- theoretical and practical explanations offered by the teacher.
+- practical activities performed by the learners.
+- debates and discussions between students and teacher or between students themselves regarding specific topics.
+- feedback from the teacher or peers.
+
+The presence and length of each of these activities during a practical session is influenced by the preference of the teacher.
```

Comment:
```suggestion
The presence and length of each of these activities during a practical session are a matter of the teacher's preference.
```

Reply:
I would reword it "Teacher preference, institutional recommendations and implementation specifics dictate:
* which types of practical sessions are used
* length of a practical session instance
* format of a practical session"